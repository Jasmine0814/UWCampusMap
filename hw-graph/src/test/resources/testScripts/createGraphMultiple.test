# test that if it allows to create 3 empty graphs

# Create a graph
created graph graph1
created graph graph2
created graph graph3

# Print the nodes in the graph
ListNodes graph1
ListNodes graph2
ListNodes graph3
