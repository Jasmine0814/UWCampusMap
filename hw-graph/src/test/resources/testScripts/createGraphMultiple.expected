# test that if it allows to create 3 empty graphs

# Create a graph
created graph graph1
created graph graph2
created graph graph3

# Print the nodes in the graph
graph1 contains:
graph2 contains:
graph3 contains:

